### DISCUSSION TITLE
"ARREW1E" Fujitsu AC remote not working with Tasmota, while working with IRremoteESP8266

### DISCUSSION THREAD

**Discussion by mattpurnell** [arendst/Tasmota#14081] [Ideas]
### PROBLEM DESCRIPTION
Tasmota on my Wemos D1 mini flashed with tasmota-ir fails to start my AC (off does work though).

BTW I absolutely love Tasmota, I will keep bashing away with this issue until I work it out :) I'm a big fan of your work.

The IR codes are collected successfully using the IRrecv, which detects it as a model 6, but sending doesn't work - other than 'off'. It is possible that my payloads aren't correct as I [struggle with the documentation](https://tasmota.github.io/docs/Tasmota-IR/) in terms of a concrete example.

When flashing the same device with IRremoteESP8266 (using their '[TurnOnFujitsuAC](https://github.com/crankyoldgit/IRremoteESP8266/blob/master/examples/TurnOnFujitsuAC/TurnOnFujitsuAC.ino)' sketch) the power-on function works, with the model set to 'ARREW4E'. This equates to Model 6 I believe?, which I have been using in the MQTT payloads.

I checked with the team at IRremoteESP8266 and they said that they requested support for my remote 8 months ago - so this may not be an 'additional support' request 

The AC model is **ASTG18K**, and the remote is **AR-REW1E**.

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [x] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [x] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [x] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [x] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [x] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [x] Device used (e.g., Sonoff Basic): Wemos D Mini
- [x] Tasmota binary firmware version number used: 10.1.0.1 
  - [x] Pre-compiled - Yes
  - [x] Self-compiled - Yes tried this too
- [x] Flashing tools used:  Tasmotiser
- [x] Provide the output of command: 
```lua
{"Vendor":"FUJITSU_AC","Model":6,"Mode":"Cool","Power":"On","Celsius":"On","Temp":19.5,"FanSpeed":"Auto","SwingV":"Off","SwingH":"Off","Quiet":"Off","Turbo":"Off","Econo":"Off","Light":"Off","Filter":"Off","Clean":"Off","Beep":"Off","Sleep":-1}
```
```
  Configuration output here:
Model set to 6, wemos has been flashed with tasmota-ir and tasmota with the USE_IR_REMOTE_FULL libraries added

```
- [x] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
  Rules output here:


```
- [x] Provide the output of this command: `Status 0`:
```lua
STATUS = {"Status":{"Module":18,"DeviceName":"loungeAC","FriendlyName":["loungeAC"],"Topic":"loungeAC","ButtonTopic":"0","Power":0,"PowerOnState":3,"LedState":1,"LedMask":"FFFF","SaveData":1,"SaveState":1,"SwitchTopic":"0","SwitchMode":[0,0,0,0,0,0,0,0],"ButtonRetain":0,"SwitchRetain":0,"SensorRetain":0,"PowerRetain":0,"InfoRetain":0,"StateRetain":0}}
STATUS1 = {"StatusPRM":{"Baudrate":115200,"SerialConfig":"8N1","GroupTopic":"aircon","OtaUrl":"http://ota.tasmota.com/tasmota/release/tasmota.bin.gz","RestartReason":"External System","Uptime":"0T00:41:28","StartupUTC":"2021-12-16T09:09:00","Sleep":50,"CfgHolder":4617,"BootCount":10,"BCResetTime":"2021-12-16T02:33:54","SaveCount":17,"SaveAddress":"3FB000"}}
STATUS2 = {"StatusFWR":{"Version":"10.1.0.1(tasmota)","BuildDateTime":"2021-12-14T23:37:33","Boot":31,"Core":"2_7_4_9","SDK":"2.2.2-dev(38a443e)","CpuFrequency":80,"Hardware":"ESP8266EX","CR":"380/699"}}
STATUS3 = {"StatusLOG":{"SerialLog":2,"WebLog":2,"MqttLog":0,"SysLog":0,"LogHost":"","LogPort":514,"SSId":["Geogaddi",""],"TelePeriod":300,"Resolution":"558180C0","SetOption":["00008009","2805C80001000600003C5A0A000000000000","00000088","00006000","00004000"]}}
STATUS4 = {"StatusMEM":{"ProgramSize":801,"Free":1244,"Heap":18,"ProgramFlashSize":4096,"FlashSize":4096,"FlashChipId":"1640C8","FlashFrequency":40,"FlashMode":3,"Features":["00000809","8FCAC787","04368001","000000CF","010013C0","C000F981","00004084","00201000","00000060"],"Drivers":"1,2,3,4,5,6,8,9,10,12,16,18,19,20,21,22,24,26,27,29,30,35,37,38,45,50","Sensors":"1,2,3,4,5,6,88"}}
STATUS5 = {"StatusNET":{"Hostname":"loungeAC-1252","IPAddress":"192.168.2.244","Gateway":"192.168.2.1","Subnetmask":"255.255.255.0","DNSServer1":"192.168.2.12","DNSServer2":"192.168.2.1","Mac":"34:AB:95:19:E4:E4","Webserver":2,"HTTP_API":1,"WifiConfig":4,"WifiPower":17.0}}
STATUS6 = {"StatusMQT":{"MqttHost":"192.168.2.12","MqttPort":1883,"MqttClientMask":"loungeAC","MqttClient":"loungeAC","MqttUser":"loungeAC","MqttCount":1,"MAX_PACKET_SIZE":1200,"KEEPALIVE":30,"SOCKET_TIMEOUT":4}}
STATUS7 = {"StatusTIM":{"UTC":"2021-12-16T09:50:28","Local":"2021-12-16T10:50:28","StartDST":"2021-10-31T03:00:00","EndDST":"2021-03-28T02:00:00","Timezone":"+01:00","Sunrise":"17:55","Sunset":"08:35"}}
STATUS10 = {"StatusSNS":{"Time":"2021-12-16T10:50:28","AM2301":{"Temperature":null,"Humidity":null,"DewPoint":null},"TempUnit":"C"}}
STATUS11 = {"StatusSTS":{"Time":"2021-12-16T10:50:28","Uptime":"0T00:41:28","UptimeSec":2488,"Heap":18,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":1,"Wifi":{"AP":1,"SSId":"Geogaddi","BSSId":"30:FD:38:F9:D2:23","Channel":6,"Mode":"11n","RSSI":82,"Signal":-59,"LinkCount":1,"Downtime":"0T00:00:03"}}}


```
- [x] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
10:53:03.726 CMD: weblog 4
10:53:03.732 MQT: stat/loungeAC/RESULT = {"WebLog":4}
10:53:03.828 WIF: Network 0, AP-, SSId vodafone70CF, Channel 1, BSSId DC:09:4C:7A:70:D8, RSSI -62, Encryption 1
10:53:03.830 WIF: Network 1, AP-, SSId DISCO Guest, Channel 1, BSSId AC:EC:85:5F:44:CC, RSSI -57, Encryption 1
10:53:03.832 WIF: Network 2, AP-, SSId DISCO, Channel 1, BSSId AC:EC:85:5F:44:C6, RSSI -57, Encryption 1
10:53:03.834 WIF: Network 3, AP1, SSId Geogaddi, Channel 6, BSSId 30:FD:38:F9:D2:23, RSSI -53, Encryption 1
10:53:03.836 WIF: Network 4, AP-, SSId Lord Voldemodem, Channel 6, BSSId B0:CC:FE:31:77:BE, RSSI -72, Encryption 1
10:53:03.838 WIF: Network 5, AP-, SSId Accio Internet, Channel 6, BSSId B0:CC:FE:31:77:BC, RSSI -72, Encryption 1
10:53:03.840 WIF: Network 6, AP1, SSId Geogaddi, Channel 11, BSSId 70:3A:CB:91:C7:EF, RSSI -62, Encryption 1
10:53:03.842 WIF: Network 7, AP-, SSId KondrHome, Channel 12, BSSId 18:F1:45:E0:E4:1E, RSSI -70, Encryption 1
10:53:04.770 CFG: Saved to flash at FD, Count 18, Bytes 4096
10:53:05.432 DHT: Timeout waiting for start signal low pulse
10:53:05.781 WIF: Checking connection...
10:53:07.430 DHT: Timeout waiting for start signal low pulse
10:53:08.483 SRC: MQTT
10:53:08.485 CMD: Grp 0, Cmnd 'IRHVAC', Idx 1, Len 244, Data '{"Vendor":"FUJITSU_AC","Model":6,"Mode":"Cool","Power":"On","Celsius":"On","Temp":19.5,"FanSpeed":"Auto","SwingV":"Off","SwingH":"Off","Quiet":"Off","Turbo":"Off","Econo":"Off","Light":"Off","Filter":"Off","Clean":"Off","Beep":"Off","Sleep":-1}'
10:53:08.645 MQT: stat/loungeAC/RESULT = {"IRHVAC":{"Vendor":"FUJITSU_AC","Model":6,"Mode":"Cool","Power":"On","Celsius":"On","Temp":19.5,"FanSpeed":"Auto","SwingV":"Off","SwingH":"Off","Quiet":"Off","Turbo":"Off","Econo":"Off","Light":"Off","Filter":"Off","Clean":"Off","Beep":"Off","Sleep":-1}}
10:53:09.407 DHT: Timeout waiting for start signal low pulse


```

### TO REPRODUCE
Send MQTT payload:
```lua
{"Vendor":"FUJITSU_AC","Model":6,"Mode":"Cool","Power":"On","Celsius":"On","Temp":19.5,"FanSpeed":"Auto","SwingV":"Off","SwingH":"Off","Quiet":"Off","Turbo":"Off","Econo":"Off","Light":"Off","Filter":"Off","Clean":"Off","Beep":"Off","Sleep":-1}
```

### EXPECTED BEHAVIOUR
I 'hoped' that the Air Con would turn on

### SCREENSHOTS
MQTT action - doesn't really benefit from screenshots..

### ADDITIONAL CONTEXT

I am more than happy to spend a lot of time troubleshooting the issue, and determining whether this model is actually supported or if there is a configuration issue on my part.

This is the 'on' captured from the remote:

```
{"IrReceived":{"Protocol":"FUJITSU_AC","Bits":128,"Data":"0x1463001010FE09315D01002304272003","Repeat":0,"IRHVAC":{"Vendor":"FUJITSU_AC","Model":6,"Mode":"Cool","Power":"On","Celsius":"On","Temp":19.5,"FanSpeed":"Auto","SwingV":"Off","SwingH":"Off","Quiet":"Off","Turbo":"Off","Econo":"Off","Light":"Off","Filter":"Off","Clean":"Off","Beep":"Off","Sleep":-1}}}
```

This is the console feedback for the successful 'on' command sent from the same device, using the IRremoteESP8266 sketch:
```
Model: 6 (ARREW4E), Id: 0, Power: On, Mode: 1 (Cool), Temp: 20C, Fan: 1 (High), 10C Heat: Off, Swing: 0 (Off), Command: N/A, Outside Quiet: Off, Timer: Off
IR Code: 0x1463001010FE0931610101000000204C
```

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by JmanNZ** [comment]
Hi
We have a Fujitsu AC with the same remote AR-REW1E and we are experiencing the same issue.
The unit only turns off.

Thanks

**Reply by sfromis** [reply]
For full IR support, use the `tasmota-ir.bin.gz` binary instead. The base build does not include all protocols.

**Reply by JmanNZ** [reply]
That's what i am using

**Reply by sfromis** [reply]
Not what mattpurnell was using, thus it is not the same issue.

**Reply by mattpurnell** [reply]
Hi, I did try the tasmota-ir build first, then moved on to a custom build later to see if there were any options I could use to influence the protocols included.

So - can confirm I also had the same issue when using the tasmota-ir build

Thanks!

**Comment by cydia2020** [comment]
I'm having the same issue, was initially thinking it's a hw problem as sending the RAW IR codes didn't work either. However after flashing the device with esphome I was able to get it to work, esphome doesn't have good Fujitsu support nonetheless so it's still a no go for me.

After flashing the device back to tasmota I tried to send some random stuff from the ir communications wiki which did work. So I'm thinking it's either a bug with the Fujitsu_AC protocol or tasmota doesn't support sending 128bit IR commands.

**Comment by trung729** [comment]
Dear [mattpurnell](https://github.com/mattpurnell),
I also have 10 Fujitsu_ac with tasmota_irhvac and have the same problem as you posted
It cannot turn on when the AC off, but I can control it with HA when I turn on with the original remote and there is 1 AC which I can turn on by pressing several times the "COOL" mode, I don't know why.
If you have solved this problem, please tell me how. 
Thanks!

**Reply by cydia2020** [reply]
I had given up on getting Fujitsu to work with our open source hardware, that being said, if you're using Home Assistant, then an alternative would be to use a Broadlink RM (I'm using the RM4, check compatibility before purchasing) with the SmartIR custom integration, it's also locally controlled and seems to work well with my Fujitsu AC.

**Reply by trung729** [reply]
I have also done that way for 1 of my AC, but each Broadlink device for only 1 command is so waste

**Reply by JmanNZ** [reply]
> Dear [mattpurnell](https://github.com/mattpurnell), I also have 10 Fujitsu_ac with tasmota_irhvac and have the same problem as you posted It cannot turn on when the AC off, but I can control it with HA when I turn on with the original remote and there is 1 AC which I can turn on by pressing several times the "COOL" mode, I don't know why. If you have solved this problem, please tell me how. Thanks!

Same i gave up and got a Sensibo works perfectly.

**Reply by mattpurnell** [reply]
> Dear [mattpurnell](https://github.com/mattpurnell), I also have 10 Fujitsu_ac with tasmota_irhvac and have the same problem as you posted It cannot turn on when the AC off, but I can control it with HA when I turn on with the original remote and there is 1 AC which I can turn on by pressing several times the "COOL" mode, I don't know why. If you have solved this problem, please tell me how. Thanks!

I ended up using IRMQTTServer from IRremoteESP8266 directly - and it works great. Last I looked Tasmota uses IRremoteESP8266. I did find a few things worth keeping in mind:

- some libraries can't handle half-degrees. So in Home Assistant or Tasmota etc. if you set a room to say 21.5 degrees, you can get no response from the IR code. I have been meaning to dig around and fix this but haven't yet - half a degree is pretty meaningless really, given you have a degree or so tolerance above and below the target temp.
- Initially I thought the matching Fujitsu 'type' within the IR code (set within the MQTT payload) was type 6. That was what Tasmota reported when listening to my remote. But Type 5 was required for controlling the Aircon. So - take the IR Listen data with a grain of salt if you are having issues.

I have been meaning to revisit Tasmota, but to include the IR Send and Receive and include some Sensor data, then I need to compile a custom image, and I haven't got around to doing that yet. Again - IRMQTTServer from IRremoteESP8266 has been so stable and useable, I have forgotten about this project. You can send IR codes direct via MQTT even after setting the ESP build to emulate a Fujitsu - so it is quite flexible, and can be used for other devices.

 https://github.com/crankyoldgit/IRremoteESP8266/blob/master/examples/IRMQTTServer/IRMQTTServer.ino

**Reply by trung729** [reply]
Many tks for your support, @mattpurnell , 
Maybe I will try to move to IRMQTTServer, 
Please show me some guidelines to add ESP8266nodeMCU after flashing arduino .ino file into home assistant , I am the beginner with arduino. I cannot find the right infomation for that.

Thanks

**Reply by trung729** [reply]
Dear All,
I have compliled IRMQTTServer, but it have this error:
--
IRMQTTServer.ino:1014:62: error: 'ac_command_t' in namespace 'stdAc' does not name a type
 1014 | String htmlSelectCommandType(const String name, const stdAc::ac_command_t def) {
--
I have searched the solution but not yet until now, please  instruct me.
Thanks

**Reply by barbudor** [reply]
This is not the best place to ask for support on other software
This software probably has it's own discussion /forum where people are more aware and will be able to help you.

**Reply by trung729** [reply]
> This is not the best place to ask for support on other software This software probably has it's own discussion /forum where people are more aware and will be able to help you.

Thanks for your advice, but I donnot know where to ask

**Reply by barbudor** [reply]
Just where you got it from
https://github.com/crankyoldgit/IRremoteESP8266/discussions
🤦

**Comment by trung729** [comment]
> ### PROBLEM DESCRIPTION
> Tasmota on my Wemos D1 mini flashed with tasmota-ir fails to start my AC (off does work though).
> 
> BTW I absolutely love Tasmota, I will keep bashing away with this issue until I work it out :) I'm a big fan of your work.
> 
> The IR codes are collected successfully using the IRrecv, which detects it as a model 6, but sending doesn't work - other than 'off'. It is possible that my payloads aren't correct as I [struggle with the documentation](https://tasmota.github.io/docs/Tasmota-IR/) in terms of a concrete example.
> 
> When flashing the same device with IRremoteESP8266 (using their '[TurnOnFujitsuAC](https://github.com/crankyoldgit/IRremoteESP8266/blob/master/examples/TurnOnFujitsuAC/TurnOnFujitsuAC.ino)' sketch) the power-on function works, with the model set to 'ARREW4E'. This equates to Model 6 I believe?, which I have been using in the MQTT payloads.
> 
> I checked with the team at IRremoteESP8266 and they said that they requested support for my remote 8 months ago - so this may not be an 'additional support' request
> 
> The AC model is **ASTG18K**, and the remote is **AR-REW1E**.
> 
> ### REQUESTED INFORMATION
> _Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_
> 
> * [x]  Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
> * [x]  Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
> * [x]  Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
> * [x]  Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
> * [x]  Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
> * [x]  Device used (e.g., Sonoff Basic): Wemos D Mini
> * [x]  Tasmota binary firmware version number used: 10.1.0.1
>   
>   * [x]  Pre-compiled - Yes
>   * [x]  Self-compiled - Yes tried this too
> * [x]  Flashing tools used:  Tasmotiser
> * [x]  Provide the output of command:
> 
> ```lua
> {"Vendor":"FUJITSU_AC","Model":6,"Mode":"Cool","Power":"On","Celsius":"On","Temp":19.5,"FanSpeed":"Auto","SwingV":"Off","SwingH":"Off","Quiet":"Off","Turbo":"Off","Econo":"Off","Light":"Off","Filter":"Off","Clean":"Off","Beep":"Off","Sleep":-1}
> ```
> 
> ```
>   Configuration output here:
> Model set to 6, wemos has been flashed with tasmota-ir and tasmota with the USE_IR_REMOTE_FULL libraries added
> ```
> 
> * [x]  If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
> 
> ```lua
>   Rules output here:
> ```
> 
> * [x]  Provide the output of this command: `Status 0`:
> 
> ```lua
> STATUS = {"Status":{"Module":18,"DeviceName":"loungeAC","FriendlyName":["loungeAC"],"Topic":"loungeAC","ButtonTopic":"0","Power":0,"PowerOnState":3,"LedState":1,"LedMask":"FFFF","SaveData":1,"SaveState":1,"SwitchTopic":"0","SwitchMode":[0,0,0,0,0,0,0,0],"ButtonRetain":0,"SwitchRetain":0,"SensorRetain":0,"PowerRetain":0,"InfoRetain":0,"StateRetain":0}}
> STATUS1 = {"StatusPRM":{"Baudrate":115200,"SerialConfig":"8N1","GroupTopic":"aircon","OtaUrl":"http://ota.tasmota.com/tasmota/release/tasmota.bin.gz","RestartReason":"External System","Uptime":"0T00:41:28","StartupUTC":"2021-12-16T09:09:00","Sleep":50,"CfgHolder":4617,"BootCount":10,"BCResetTime":"2021-12-16T02:33:54","SaveCount":17,"SaveAddress":"3FB000"}}
> STATUS2 = {"StatusFWR":{"Version":"10.1.0.1(tasmota)","BuildDateTime":"2021-12-14T23:37:33","Boot":31,"Core":"2_7_4_9","SDK":"2.2.2-dev(38a443e)","CpuFrequency":80,"Hardware":"ESP8266EX","CR":"380/699"}}
> STATUS3 = {"StatusLOG":{"SerialLog":2,"WebLog":2,"MqttLog":0,"SysLog":0,"LogHost":"","LogPort":514,"SSId":["Geogaddi",""],"TelePeriod":300,"Resolution":"558180C0","SetOption":["00008009","2805C80001000600003C5A0A000000000000","00000088","00006000","00004000"]}}
> STATUS4 = {"StatusMEM":{"ProgramSize":801,"Free":1244,"Heap":18,"ProgramFlashSize":4096,"FlashSize":4096,"FlashChipId":"1640C8","FlashFrequency":40,"FlashMode":3,"Features":["00000809","8FCAC787","04368001","000000CF","010013C0","C000F981","00004084","00201000","00000060"],"Drivers":"1,2,3,4,5,6,8,9,10,12,16,18,19,20,21,22,24,26,27,29,30,35,37,38,45,50","Sensors":"1,2,3,4,5,6,88"}}
> STATUS5 = {"StatusNET":{"Hostname":"loungeAC-1252","IPAddress":"192.168.2.244","Gateway":"192.168.2.1","Subnetmask":"255.255.255.0","DNSServer1":"192.168.2.12","DNSServer2":"192.168.2.1","Mac":"34:AB:95:19:E4:E4","Webserver":2,"HTTP_API":1,"WifiConfig":4,"WifiPower":17.0}}
> STATUS6 = {"StatusMQT":{"MqttHost":"192.168.2.12","MqttPort":1883,"MqttClientMask":"loungeAC","MqttClient":"loungeAC","MqttUser":"loungeAC","MqttCount":1,"MAX_PACKET_SIZE":1200,"KEEPALIVE":30,"SOCKET_TIMEOUT":4}}
> STATUS7 = {"StatusTIM":{"UTC":"2021-12-16T09:50:28","Local":"2021-12-16T10:50:28","StartDST":"2021-10-31T03:00:00","EndDST":"2021-03-28T02:00:00","Timezone":"+01:00","Sunrise":"17:55","Sunset":"08:35"}}
> STATUS10 = {"StatusSNS":{"Time":"2021-12-16T10:50:28","AM2301":{"Temperature":null,"Humidity":null,"DewPoint":null},"TempUnit":"C"}}
> STATUS11 = {"StatusSTS":{"Time":"2021-12-16T10:50:28","Uptime":"0T00:41:28","UptimeSec":2488,"Heap":18,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":1,"Wifi":{"AP":1,"SSId":"Geogaddi","BSSId":"30:FD:38:F9:D2:23","Channel":6,"Mode":"11n","RSSI":82,"Signal":-59,"LinkCount":1,"Downtime":"0T00:00:03"}}}
> ```
> 
> * [x]  Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
> 
> ```lua
> 10:53:03.726 CMD: weblog 4
> 10:53:03.732 MQT: stat/loungeAC/RESULT = {"WebLog":4}
> 10:53:03.828 WIF: Network 0, AP-, SSId vodafone70CF, Channel 1, BSSId DC:09:4C:7A:70:D8, RSSI -62, Encryption 1
> 10:53:03.830 WIF: Network 1, AP-, SSId DISCO Guest, Channel 1, BSSId AC:EC:85:5F:44:CC, RSSI -57, Encryption 1
> 10:53:03.832 WIF: Network 2, AP-, SSId DISCO, Channel 1, BSSId AC:EC:85:5F:44:C6, RSSI -57, Encryption 1
> 10:53:03.834 WIF: Network 3, AP1, SSId Geogaddi, Channel 6, BSSId 30:FD:38:F9:D2:23, RSSI -53, Encryption 1
> 10:53:03.836 WIF: Network 4, AP-, SSId Lord Voldemodem, Channel 6, BSSId B0:CC:FE:31:77:BE, RSSI -72, Encryption 1
> 10:53:03.838 WIF: Network 5, AP-, SSId Accio Internet, Channel 6, BSSId B0:CC:FE:31:77:BC, RSSI -72, Encryption 1
> 10:53:03.840 WIF: Network 6, AP1, SSId Geogaddi, Channel 11, BSSId 70:3A:CB:91:C7:EF, RSSI -62, Encryption 1
> 10:53:03.842 WIF: Network 7, AP-, SSId KondrHome, Channel 12, BSSId 18:F1:45:E0:E4:1E, RSSI -70, Encryption 1
> 10:53:04.770 CFG: Saved to flash at FD, Count 18, Bytes 4096
> 10:53:05.432 DHT: Timeout waiting for start signal low pulse
> 10:53:05.781 WIF: Checking connection...
> 10:53:07.430 DHT: Timeout waiting for start signal low pulse
> 10:53:08.483 SRC: MQTT
> 10:53:08.485 CMD: Grp 0, Cmnd 'IRHVAC', Idx 1, Len 244, Data '{"Vendor":"FUJITSU_AC","Model":6,"Mode":"Cool","Power":"On","Celsius":"On","Temp":19.5,"FanSpeed":"Auto","SwingV":"Off","SwingH":"Off","Quiet":"Off","Turbo":"Off","Econo":"Off","Light":"Off","Filter":"Off","Clean":"Off","Beep":"Off","Sleep":-1}'
> 10:53:08.645 MQT: stat/loungeAC/RESULT = {"IRHVAC":{"Vendor":"FUJITSU_AC","Model":6,"Mode":"Cool","Power":"On","Celsius":"On","Temp":19.5,"FanSpeed":"Auto","SwingV":"Off","SwingH":"Off","Quiet":"Off","Turbo":"Off","Econo":"Off","Light":"Off","Filter":"Off","Clean":"Off","Beep":"Off","Sleep":-1}}
> 10:53:09.407 DHT: Timeout waiting for start signal low pulse
> ```
> 
> ### TO REPRODUCE
> Send MQTT payload:
> 
> ```lua
> {"Vendor":"FUJITSU_AC","Model":6,"Mode":"Cool","Power":"On","Celsius":"On","Temp":19.5,"FanSpeed":"Auto","SwingV":"Off","SwingH":"Off","Quiet":"Off","Turbo":"Off","Econo":"Off","Light":"Off","Filter":"Off","Clean":"Off","Beep":"Off","Sleep":-1}
> ```
> 
> ### EXPECTED BEHAVIOUR
> I 'hoped' that the Air Con would turn on
> 
> ### SCREENSHOTS
> MQTT action - doesn't really benefit from screenshots..
> 
> ### ADDITIONAL CONTEXT
> I am more than happy to spend a lot of time troubleshooting the issue, and determining whether this model is actually supported or if there is a configuration issue on my part.
> 
> This is the 'on' captured from the remote:
> 
> ```
> {"IrReceived":{"Protocol":"FUJITSU_AC","Bits":128,"Data":"0x1463001010FE09315D01002304272003","Repeat":0,"IRHVAC":{"Vendor":"FUJITSU_AC","Model":6,"Mode":"Cool","Power":"On","Celsius":"On","Temp":19.5,"FanSpeed":"Auto","SwingV":"Off","SwingH":"Off","Quiet":"Off","Turbo":"Off","Econo":"Off","Light":"Off","Filter":"Off","Clean":"Off","Beep":"Off","Sleep":-1}}}
> ```
> 
> This is the console feedback for the successful 'on' command sent from the same device, using the IRremoteESP8266 sketch:
> 
> ```
> Model: 6 (ARREW4E), Id: 0, Power: On, Mode: 1 (Cool), Temp: 20C, Fan: 1 (High), 10C Heat: Off, Swing: 0 (Off), Command: N/A, Outside Quiet: Off, Timer: Off
> IR Code: 0x1463001010FE0931610101000000204C
> ```
> 
> **(Please, remember to close the issue when the problem has been addressed)**

I have also flashed my ESP-01M with ESPHome but it was still not been turned on
https://vi.aliexpress.com/item/1005004486068078.html?spm=a2g0o.productlist.main.1.2173122bQU3zn4&algo_pvid=93120b54-9488-4e7c-aa20-72d817045381&algo_exp_id=93120b54-9488-4e7c-aa20-72d817045381-0&pdp_npi=3%40dis%21VND%2180481.0%2171565.0%21%21%21%21%21%402102196716830349051204220d077d%2112000029327246562%21sea%21VN%210&curPageLogUid=D0pzuNqT7AqY

**Reply by mattpurnell** [reply]
See my response to an earlier reply above - I went with IRMQTTServer from IRremoteESP8266 and it has been great.
